From 3e2561b2dd8a7f40499ed3b3e0638bb9332441c6 Mon Sep 17 00:00:00 2001
From: Sachin Prabhu <sprabhu@redhat.com>
Date: Mon, 28 Jul 2014 12:16:54 -0400
Subject: [fs] cifs: new helper file_inode(file)

Message-id: <1406549822-10376-62-git-send-email-sprabhu@redhat.com>
Patchwork-id: 86785
O-Subject: [RHEL7.1 PATCH 61/69] [bz1082049]fs: cifs: new helper: file_inode(file)
Bugzilla: 1082049
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Libo Chen <clbchenlibo.chen@huawei.com>

Signed-off-by: Libo Chen <clbchenlibo.chen@huawei.com>
Acked-by: Jeff Layton <jlayton@redhat.com>
Signed-off-by: Steve French <smfrench@gmail.com>
(cherry picked from commit 2d4f84bd796e75608a18805255398dd9010e5c99)
Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/cifs/ioctl.c b/fs/cifs/ioctl.c
index 7749230..45cb59b 100644
--- a/fs/cifs/ioctl.c
+++ b/fs/cifs/ioctl.c
@@ -85,7 +85,7 @@ static long cifs_ioctl_clone(unsigned int xid, struct file *dst_file,
   goto out_fput;
  }
 
- src_inode = src_file.file->f_dentry->d_inode;
+ src_inode = file_inode(src_file.file);
 
  /*
   * Note: cifs case is easier than btrfs since server responsible for
-- 
1.7.1